child_compassion/models/compassion_hold.py

```diff
@@ -9,12 +9,15 @@
 #
 ##############################################################################
 
+from openerp.addons.message_center_compassion.mappings \
+    import base_mapping as mapping

```

Comment:
Instead of using base_mapping you can directly import the correct mapping :
```
from ..mappings.child_reinstatement_mapping import ReinstatementMapping
```